plugins/woocommerce/src/Utilities/NumberUtil.php

```diff
@@ -49,4 +53,66 @@ public static function array_sum( array $arr ): float {
 
 		return array_sum( $sanitized_array );
 	}
+
+	/**
+	 * Sanitize a cost value based on the current locale decimal and thousand separators.
+	 *
+	 * @param string $value               The value to sanitize.
+	 * @return string The sanitized value, or empty string if invalid.
```

Comment:
If `$value` is invalid, it will throw an exception.

The suggestion below also fixes indentation.

```suggestion
	 * @return string                     The sanitized value.
```